src/main/java/com/ldtteam/structurize/client/gui/WindowScan.java

```diff
@@ -317,11 +325,6 @@ private void updateResources()
         resources.clear();
         entities.clear();
 
-        if (findPaneByID(BUTTON_SHOW_RES).isVisible())
```

Comment:
Hmm?

Reply:
ye just saw it too, didnt mean to remove that